internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixCustomizedViewOfflinePushRepository.java

```diff
@@ -52,7 +51,44 @@ public HelixCustomizedViewOfflinePushRepository(SafeHelixManager manager, ReadOn
     super(manager);
     dataSource.put(PropertyType.CUSTOMIZEDVIEW, Collections.singletonList(HelixPartitionState.OFFLINE_PUSH.name()));
     this.storeRepository = storeRepository;
-    this.storeRepository.registerStoreDataChangedListener(this);
+    this.storeRepository.registerStoreDataChangedListener(new StoreDataChangedListener() {
```

Comment:
Hmm... I guess this is just a nitpick, but why move the code from normal class functions to an anonymous class here? The previous form looked a bit cleaner IMHO. Anonymous classes make stacktraces a bit harder to interpret... is there a benefit I'm not thinking of?

Reply:
Converted it to inner class as discussed offline.